File: 1008.cpp
// André Nascimento
// GitHub: github.com/AndreNasci
// Linkedin: linkedin.com/in/andre-nascimento-a01095185

#include <bits/stdc++.h>

using namespace std;

int main()
{
    int worked_hours, number;
    double salary, per_hour;

    cin >> number;
    cin >> worked_hours;
    cin >> per_hour;

    salary = worked_hours * per_hour;

    cout << "NUMBER = " << number << endl;
    cout << "SALARY = U$ " << fixed << setprecision(2) << salary << endl;
    return 0;
}

File: Python/1008.py
"""
André Nascimento
GitHub: github.com/AndreNasci
Linkedin: linkedin.com/in/andre-nasci/
"""

employee_number = int(input())
worked_hours = int(input())
amount_per_hour = float(input())

salary = worked_hours * amount_per_hour

print("NUMBER =", employee_number)
print(f"SALARY = U$ {salary:.2f}")